src/js/index.js

```diff
@@ -0,0 +1,191 @@
+            </div>
+            <div class="col-4 col-md-3 col-lg-4">
+              <label>{{ player.guest_name }}</label>
+            </div>
+            <div class="col-4 col-md-3">
+              <input class="form-control" type="text" v-model="player.guest_position" />
+            </div>
+            <div class="col-2 col-md-1">
+              <input class="form-control" type="checkbox" v-model="player.guest_paid"/>
+            </div>
+            <div class="col-12 col-md-4 col-lg-3 text-right text-md-left">
+              <div class="mt-3 d-block d-md-none"></div>
+              <input type="button" class="btn btn-success" value="Update" @click="update_guest(player)"/>
+              <input type="button" class="btn btn-danger" value="Remove" @click="remove_guest(player)"/>
+            </div>
+          </div>
+        </div>
+        <hr class="bold" />
+        <div class="row form-inline">
+          <div class="col-2 col-md-1"><label><b>New</b></label></div>
+          <div class="col-4 col-md-3 col-lg-4">
+            <input class="form-control" value="" placeholder="Name" v-model="new_player.name"/>
+          </div>
+          <div class="col-4 col-md-3">
+            <input class="form-control" type="text" v-model="new_player.position" />
+          </div>
+          <div class="col-2 col-md-1">
+            <input class="form-control" type="checkbox" v-model="new_player.paid" disabled />
+          </div>
+          <div class="col-12 col-md-4 col-lg-3 text-right text-md-left">
+            <div class="mt-3 d-block d-md-none"></div>
+            <input type="button" class="btn btn-success" value="Add player" @click="add_guest" />
+          </div>
+        </div>
+      </div>`,
+    data: {
+      players: [],
+      new_player: {},
+    },
+    methods: {
+      add_guest: function () {
+        action_reload({
+          action: 'add_guest',
+          event_id: get_query_value('event_id'),
+          name: this.new_player.name,
+          position: this.new_player.position,
+        });
+      },
+      update_guest: function (player) {
+        if (confirm('Are you sure you want to update ' + player.guest_name)) {
+
+          action_reload({
+            action: 'update_guest',
+            id: player.guest_id,
+            position: player.guest_position,
+            is_paid: player.guest_paid
+          });
+        }
+      },
+      remove_guest: function (player) {
+        var p = prompt('Type yes to remove ' + player.guest_name + '. This is permanent!');
+        if (p == null) {
+          return;
+        }
+        if (p.toUpperCase() != 'YES') {
+          alert('Remove failed. You had to type yes to remove.');
+          return;
+        }
+        action_reload({
+          action: 'remove_guest',
+          id: player.guest_id
+        });
+      }
+    }
+  })
+
+  const currentEventInfo = new Vue({
+    el: '#current-event-info',
+    template: `
+      <p>
+        Most recent event:
```

Comment:
`Most recent event` was gone already, by the way

Reply:
removed